backend/app/database/crud.py

```diff
@@ -26,20 +17,26 @@ def delete_team(db: Session, team_id: int):
     db.delete(to_remove)
     db.commit()
 
-# Players:
+def update_team(db: Session, team_id: int, team: schemas.TeamUpdate):
+    db_team = db.query(models.Team).filter(models.Team.id == team_id).first()
```

Comment:
I think you should add a handler that will return an error if query will return `None`.
You can also try using try/except with `one()`, instead of using `first()` up to you.